### ISSUE TITLE
Steering committee election

### ISSUE THREAD

**Issue by kcranston** [carpentries/2017Merger#12]
List of things that need to be done for the upcoming election. 

- [x] @maneesha mail list of voters to @kcranston
- [ ] @kcranston set up ElectionBuddy
- [x] after Nov 30: @kcranston email candidates re: procedures (as suggested by @k8hertweck , the two candidates with the first and second most votes serve two year terms, while those with the third and fourth most votes serve one year terms)
- [x] after Nov 30: @k8hertweck to write blog post with links to candidate bios, post to DC and SWC blogs
- [x] @kcranston to open and close ElectionBuddy poll. As suggested by @maneesha: open at midnight between Dec 3 and 4 in time zone UTC+12  (the first place in the world it becomes Dec 4) and end at midnight between Dec 8 and 9 in time zone UTC -12 (the last place it's still Dec 8)
- [x] @kcranston emailed discuss list letting people know election open
- [ ] after Dec 8: @kcranston export results from ElectionBuddy and provide to staff and steering committee
- [ ] contact all candidates with results
- [ ] report results publicly on DC and SWC blog

**Comment by maneesha** [comment]
Thanks for summarizing our next steps, @kcranston 

Question for @k8hertweck -- do the nominees know about this suggestion for allocating votes to terms?  I think everyone may have thought they were running for a one year term.

**Comment by k8hertweck** [comment]
@maneesha No, we haven't specified how we'd handle the peculiarities of this first round of voting until it came time to put the ballot together. I had assumed all candidates would expect a two year term, as that is what is described in the [RFC](https://github.com/carpentries/2017Merger/blob/master/RFCs/RFC2.md), but I see that the [announcement blog post](https://software-carpentry.org/blog/2017/10/call-new-joint-board.html) wasn't explicit about the maximum term. This should be explained when emailing candidates as well. Who wants to handle this?

**Comment by kcranston** [comment]
I've volunteered to send the email to the candidates with this info.

**Comment by weaverbel** [comment]
I have done a blog post that links to all 8 nominees so far to make it easier for you http://www.datacarpentry.org/blog/meet-candidates/ @k8hertweck @kcranston

**Comment by kcranston** [comment]
Just for my own notes:

00:00 Dec 4 UTC+12 = 07:00 Sun Dec 3 EST
11:59 Dec 8 UTC-12 = 06:59 Sat Dec 9 EST

Election set up and notices will go out starting at 00:00 UTC+12 Dec 4.

**Comment by weaverbel** [comment]
@kcranston Do we have a date when results will be announced? Asking for newsletter.

**Comment by kcranston** [comment]
Likely sometime Monday Dec 11.

**Comment by weaverbel** [comment]
Thanks @kcranston !

**Comment by maneesha** [comment]
I want to document this whole process for next time.  Some notes for reference for here: 
1.  We should have had someone specifically designated to check for submitted nominations and post them on the blog until UTC+12 on the last day of nominations (the last time zone in the world that it is still that day)

2.  We had a lot of trouble with the query used to generate the member list.  This is the query we used

**Comment by maneesha** [comment]
```
select contact from
(
select person.personal || ' ' || person.family || ' ' || person.email as contact

from workshops_person as person
join workshops_event as event
join workshops_task as task
join workshops_role as role
join workshops_award as award
join workshops_badge as badge
join workshops_event_tags as event_tags
join workshops_tag as tag

on person.id = task.person_id
and event.id = task.event_id
and role.id = task.role_id
and award.person_id = person.id
and award.badge_id = badge.id
and event_tags.event_id = event.id
and event_tags.tag_id = tag.id

where role.name = 'instructor'
and badge.name in ('swc-instructor','dc-instructor', 'trainer')
and (tag.name = 'LC' or tag.name = 'DC' or tag.name = 'SWC' or tag.name = 'TTT')
and event.start >= '2015-11-30'
and event.end <= '2017-12-01'

group by person.id
having count(*) > 1

union

select person.personal || ' ' || person.family || ' ' || person.email || ' ' as contact
from workshops_person as person
join workshops_award as award
join workshops_badge as badge

on award.person_id = person.id
and award.badge_id = badge.id

where badge.name = 'member'

);

```

**Comment by maneesha** [comment]
3.  We need to update this page (https://software-carpentry.org/scf/members/).  This has not been done yet.  Also need to document how this page is built.
